Kickstarter-iOS/Locales/Base.lproj/Localizable.strings

```diff
@@ -4,7 +4,7 @@
 "Account" = "Account";
 "Add" = "Add";
 "Add_attachments" = "Add attachments…";
-"Add_new_card" = "Add a new card";
```

Comment:
Looks like adding all of the strings unrelated to this cell make this PR a hefty one (strings + screenshot updates that should require careful QA outside of the scope of the description cell). 

Would it make sense to leave these to other work?

Reply:
I think it's fine as it is for now.